global.json

```
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "7.0.100-rc.1.22431.12",
+    "dotnet": "8.0.100-alpha.1.22513.4",
```

It passes on my machine when I update the SDK:
```suggestion
    "dotnet": "8.0.100-alpha.1.22514.14",
```

---

Thanks. Sounds like it's a difference between the SDK built and the SDK used for building (we call it the stage 0 SDK). Let's try updating that.